src/page-modules/assistant/types.ts

```diff
@@ -30,8 +30,12 @@ export const TripQuerySchema = z.object({
   toLat: z.number(),
   toLayer: z.union([z.literal('address'), z.literal('venue')]),
   filter: z.string().optional(),
-  departureDate: z.number().optional(),
-  departureMode: z.nativeEnum(DepartureMode),
+  searchMode: z.union([
+    z.literal('now'),
+    z.literal('arriveBy'),
+    z.literal('departBy'),
```

Comment:
Not sure if it's possible or that we should, but could it be wise to use the const SEARCH_MODES here? To avoid having to update the values two places

Reply:
<img width="1047" alt="2023-11-09 at 15 18 18@2x" src="https://github.com/AtB-AS/planner-web/assets/14045515/8f5bc2ed-bf03-405b-bef6-8c3e79d1061b">


Seems like we can't

Reply:
Can specify it originally as zod object and infer type from that and use that in reference

Reply:
Good idea. So we'll end up having `SEARCH_MODES` (for mapping through in the component, maybe not necessary, could just use an array there), the zod type, and inferred type from zod type.

Reply:
Pushed changes to that now, [18341a3](https://github.com/AtB-AS/planner-web/pull/58/commits/18341a39ed232f86e2fe802900e4e1849b5fb6f8)